apps/server/test/lexical/convertibles/lexical.plugin.diagnostic.result_test.exs

```diff
@@ -20,7 +20,7 @@ defmodule Lexical.Convertibles.Lexical.Plugin.V1.Diagnostic.ResultTest do
         def fun_two do
           "🎸hello"
         end
-      end
+      endpl
```

Comment:
Typo?

Reply:
I was thinking: how did the tests even pass with this?